pages/specification/migration/_index.md

```diff
@@ -3,5 +3,78 @@ title: Migrating from older drafts
+| items                 | Applicator             | 2019-09            | No      | Updated to prefixItems        |
+| maxContains           | Validation             | 2019-09            | No      | No                            |
+| maxProperties         | Validation             | 04                 | No      | No                            |
+| maximumCanEqual       | Validation             | 02                 | No      | Updated to exclusiveMaximum   |
+| minimumCanEqual       | Validation             | 02                 | No      | Updated to exclusiveMinimum   |
+| minContains           | Validation             | 2019-09            | No      | No                            |
+| minProperties         | Validation             | 04                 | No      | No                            |
+| multipleOf            | Validation             | 04                 | No      | Updated from divisibleBy      |
+| not                   | Validation             | 04                 | No      | No                            |
+| oneOf                 | Validation             | 04                 | No      | No                            |
+| optional              | Validation             | 02                 | No      | Updated to (required)         |
+| pattern               | Core                   | 03                 | No      | No                            |
+| patternProperties     | Validation             | 03                 | No      | No                            |
+| prefixItems           | Applicator             | 2020-12            | No      | Updated from items            |
+| propertyNames         | Validation             | 06                 | No      | No                            |
+| readOnly              | Validation             | 07                 | No      | No                            |
+| required              | Validation             | 03                 | No      | No                            |
+| requires              | Validation             | 02                 | Yes     | -                             |
+| title                 | Annotation             | 01                 | No      | No                            |
+| then                  | Validation             | 07                 | No      | No                            |
+| type                  | Validation             | 02                 | No      | No                            |
+| unevaluatedItems      | Applicator, Annotation | 2019-09            | No      | No                            |
+| unevaluatedProperties | Applicator, Annotation | 2019-09            | No      | No                            |
+| uniqueItems           | Assertion              | 02                 | Yes     | No                            |
+| writeOnly             | Validation             | 07                 | No      | No                            |
+
+For a detailed read-through about all the changes see each Draft migration guide.
```

Comment:
Maybe link to them here?

Reply:
Yes, they are yet to be available. I will link them once they're available.